### PR TITLE
All files of javascript2-week3 homework

### PR DESCRIPTION
**Pull request by sayali05**

### CODE REVIEW

File: javascript/javascript2/week3/fastest-presser.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		infoTag.innerHTML = "Please enter right key";
+	}
+}
+function startGame() {
+	let time = durationInputTag.value;
+	if (time === "") {
+		infoTag.innerHTML = "Please enter time to start game";
+	} else {
+		infoTag.innerHTML =
+			"Game started now and Time limit is set for " + time + " seconds";
```
Comment (naresh699) [inline javascript/javascript2/week3/fastest-presser.js:29]: try writing 'template string' if code is from scratch.

File: javascript/javascript2/week3/geolocation.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const pTag = document.getElementById("result");
+
+buttonTag.addEventListener("click", () => {
+	navigator.geolocation.getCurrentPosition(success, error);
+});
+function success(position) {
+	const latitude = position.coords.latitude;
+	const longitude = position.coords.longitude;
+
+	pTag.innerHTML = `This is the latitude: ${latitude}. <br> This is the longitude: ${longitude}.`;
```
Comment (naresh699) [inline javascript/javascript2/week3/geolocation.js:15]: good use of template string, use this always if code is in your control and fresh

File: javascript/javascript2/week3/warmup.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+//1.Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+setTimeout(function () {
+	console.log("Called after 2.5 seconds after the script is loaded");
+}, 2500);
+
+//2.Create a function that takes 2 parameters: delay and stringToLog.
```
Comment (naresh699) [inline javascript/javascript2/week3/warmup.js:7]: I see huge number of line for 'commenting only', this is totally fine. but better coding practice please read about 'JSDoc'. check some youtbe tutorials.

**Review by naresh699 — APPROVED** [review]
nice work. 
1. i am seeing some extra formating , assuming tools is making this format.
2. read about JSDoc when you have time, this will be real advantage if you sit in any interview

File: javascript/javascript2/week3/warmup.js (hunk 3 above)
Comment (sayali05) [reply javascript/javascript2/week3/warmup.js:7]: Thank you for the review. i will read 'JSDoc' and update my code.

**Pull request merged**